src/js/base/BoxGroupSectionSelect.ts

```diff
@@ -1,17 +1,16 @@
 import styled, { DefaultTheme } from "styled-components";
-import { BoxGroupSection } from "./BoxGroupSection";
+import { BoxGroupSection, BoxGroupSectionProps } from "./BoxGroupSection";
 import { CheckboxLabel } from "./CheckboxLabel";
 import { StyledCheckbox } from "./styled/StyledCheckbox";
 
-type SelectBoxGroupSectionProps = {
+type SelectBoxGroupSectionProps = BoxGroupSectionProps & {
```

Comment:
Hmm feel like it makes more sense to either keep them separate or fuse them fully. Everything in `SelectBoxGroupsSecctionProps` is already in `BoxGroupSectionProps`